Example/Assets/en.lproj/Localizable.strings

```diff
@@ -1,32 +1,24 @@
-"error_title" = "Error";
-"error_list_message" = "Couldn't retrieve Miniapp list, please try again later";
-"error_single_message" = "Couldn't retrieve Miniapp info, please try again later";
-"error_no_published_version" = "No published versions for the provided Mini App ID.";
-"error_miniapp_id_not_found" = "No Mini App found for the provided Mini App ID.";
-"error_incorrect_appid_message" = "Incorrect Host app ID, please try again";
-"error_empty_appid_key_message" = "Host app ID cannot be empty, enter valid ID and try again";
-"error_incorrect_subscription_key_message" = "Incorrect subscription key, please try again";
-"error_empty_subscription_key_message" = "Subscription key cannot be empty, enter valid subscription key and try again";
-"error_miniapp_message" = "Couldn't retrieve Miniapp, please try again later";
-"error_miniapp_download_message" = "Downloading failed, please try again later";
-"error_invalid_miniapp_id" = "Invalid Miniapp ID";
-"error_no_miniapp_found" = "No Miniapp found, please contact your administrator";
```

Comment:
Well, I specifically put all these deleted strings in documentation ;) Also, there is the Confluence page for Link localizables that I will update when/if the PR is validated

Reply:
After thinking twice I realize what you're talking about... That's true that for downward compatibility we should keep them, but now our focus has changed, it may not be an issue (cc: @corycaywood)

Reply:
I think we just forgot a fact, it's that these localizable were not documented until now, so it should be fine to drop these keys